examples/haskell-flake-project/build.nix

```diff
@@ -16,6 +16,15 @@ _: {
     in
 
     {
+      packages = {
+        haskell-flake-project-src = pkgs.stdenv.mkDerivation {
+          name = "haskell-flake-project-src";
+          src = ./.;
+          phases = "installPhase";
+          installPhase = "ln -s $src $out";
```

Comment:
I am very surprised this works. Usually we just do a `cp`.

Reply:
maybe we could expose this by default from the haskellFlake, like we do with Rust

Reply:
No idea! But my hunch is that if `ln` is not safe, it wouldn't be here?

@szg251 do eeeeeeeet